src/main/java/dao/FlightDAO.java

```diff
@@ -1,33 +1,88 @@
 package dao;
 
 import entities.Flight;
+import services.ConnectionFactory;
 
-import java.sql.SQLException;
+import java.sql.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashSet;
 import java.util.Set;
 
 public class FlightDAO implements DAO<Flight> {
+
+    private Flight extractFlightFromResultSet(ResultSet resultSet) throws SQLException {
+        Flight flight = new Flight();
+        flight.setFlightId( resultSet.getInt("idFlight") );
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        flight.setDateTime(LocalDateTime.from(dateTimeFormatter.parse(resultSet.getString("DateTime").replaceAll("\\.0",""))));
```

Comment:
too many expressions in one line... it's better to make some variables to increase readability